### DISCUSSION TITLE
How to remove final html screen at the end of the experiment? cognition.run

### DISCUSSION THREAD

**Discussion by davidebottari** [jspsych/jsPsych#3117] [Q&A]
Hi all, 
We have an issue with each experiment we run on cognition.run
Probably this a very stupid issue! :)
At the end of the experiment, we would like a blank screen and not an HTML one. 
Any can help us with how we can do this? 
I attach below the code of one of the experiments (based on the example tasks provided in jsPsych). 
Thanks in advance!!
Best, 
Davide

var jsPsych = initJsPsych({
      on_finish: function() {
        jsPsych.data.displayData();
      }
    });

    /* experiment parameters */
    var reps_per_trial_type = 4;

    /* Code */
    var Code = {
  type: jsPsychSurveyText,
  button_label: ['Continua'],
  questions: [
    {prompt: 'Inserisci il Codice Alfanumerico Personale già usato in fase di accesso alla survey.', required: true}
  ]
};

    /*set up welcome block*/
    var welcome = {
      type: jsPsychHtmlKeyboardResponse,
      stimulus: "Ti diamo il benvenuto all’esperimento! Premi un tasto qualsiasi per poter iniziare."
    };

    /*set up instructions block*/
    var instructions = {
      type: jsPsychHtmlKeyboardResponse,
      stimulus: "<p  align=left>In questo compito, vedrai sullo schermo cinque frecce, così come mostrato nell'esempio seguente</p>"+
        "<img src='img/inc1.png'></img>"+
        "<p align=left>Dovrai indicare la direzione della freccia centrale. Fornirai le tue risposte a questo compito utilizzando la tastiera del tuo PC. Premi il tasto freccia sinistra se la freccia centrale sta puntando a sinistra (<).  Premi il tasto freccia destra se la freccia centrale sta puntando a destra (>). Premi qualsiasi tasto per iniziare. </p>",
      post_trial_gap: 1000
    };

    /*defining stimuli*/
    var test_stimuli = [
      {
        stimulus: "img/con1.png",
        data: { stim_type: 'congruent', direction: 'left'}
      },
      {
        stimulus: "img/con2.png",
        data: { stim_type: 'congruent', direction: 'right'}
      },
      {
        stimulus: "img/inc1.png",
        data: { stim_type: 'incongruent', direction: 'right'}
      },
      {
        stimulus: "img/inc2.png",
        data: { stim_type: 'incongruent', direction: 'left'}
      }
    ];

    /* defining test timeline */
    var test = {
      timeline: [{
        type: jsPsychImageKeyboardResponse,
        choices: ['ArrowLeft', 'ArrowRight'],
        trial_duration: 1500,
        stimulus: jsPsych.timelineVariable('stimulus'),
        data: jsPsych.timelineVariable('data'),
        on_finish: function(data) {
          var correct = false;
          if (data.direction == 'left' && jsPsych.pluginAPI.compareKeys(data.response, 'ArrowLeft') && data.rt > -1) {
            correct = true;
          } else if (data.direction == 'right' && jsPsych.pluginAPI.compareKeys(data.response, 'ArrowRight') && data.rt > -1) {
            correct = true;
          }
          data.correct = correct;
        },
        post_trial_gap: function() {
            return Math.floor(Math.random() * 1500) + 500;
        }
      }],
      timeline_variables: test_stimuli,
      sample: {type: 'fixed-repetitions', size: reps_per_trial_type}
    };

    // manually preload images due to presenting them with timeline variables
    var images = ["img/con1.png","img/con2.png","img/inc1.png","img/inc2.png"];
    var preload = {
      type: jsPsychPreload,
      images: images
    };

    /*set up experiment structure*/
    var timeline = [];
    timeline.push(preload);
    timeline.push(Code);
    timeline.push(welcome);
    timeline.push(instructions);
    timeline.push(test);

    /*start experiment*/
    jsPsych.run(timeline);

**Comment by jodeleeuw** [comment]
Hi @davidebottari,

This is probably a question best aimed at the cognition.run team if the site is adding a default screen at the end of the experiment.
